src/common/gateway/gateway.service.ts

```diff
@@ -133,6 +135,12 @@ export class GatewayService {
     return new NftData(result.tokenData);
   }
 
+  async getTransactionPool(): Promise<TransactionInPool[]> {
+    const result = await this.get(`transaction/pool?fields=nonce,sender,receiver,gaslimit,gasprice,receiverusername,data,value`, GatewayComponentRequest.transactionPool);
+
+    return result;
```

Comment:
You can use direct return in this case `return result ...`

Reply:
done